app/models/metric/capture.rb

```diff
@@ -20,6 +20,10 @@ def self.historical_start_time
     historical_days.days.ago.utc.beginning_of_day
   end
 
+  def self.targets_archived_from
+    Settings.performance.targets.archived_for.to_i_with_method.seconds.ago.utc
```

Comment:
too.long.chain.didnt.follow :)
do you mind splitting `.seconds.ago.utc` to another statement?

Reply:
done